src/main/resources/hudson/plugins/plot/PlotReport/index.jelly

```diff
@@ -51,6 +51,11 @@
           </div>
         </j:if>
         <img src="getPlot?index=${index}&amp;width=750&amp;height=450" width="750" height="450" lazymap="getPlotMap?index=${index}" />
+        <div style="width:750px">
+          <j:if test="${it.getPlotDescription(index) != null}">
```

Comment:
You should be checking here not only for `null` but for String emptiness as well.
```
${it.getPlotDescription(index) != null && !it.getPlotDescription(index).isEmpty()}
```

Reply:
This line was failing. The `&&` operation was failing during the tests

possible solution:
```
          <j:if test="${it.getPlotDescription(index) != null}">
            <j:if test="${!it.getPlotDescription(index).isEmpty()}">
              <b>Description</b>: <j:out value="${it.getPlotDescription(index)}"/>
            </j:if>
          </j:if>
```

Reply:
Well, it works this way, but fine 😅 
```
<j:if test="${it.getPlotDescription(index) != null &amp;&amp; !it.getPlotDescription(index).isEmpty()}">
```